src/app/pages/token/token-detail/token-overview/token-overview.component.ts

```diff
@@ -80,4 +98,20 @@ export class TokenOverviewComponent implements OnInit {
       this.tokenDetail['holder'] = res.view_count_holder_cw721_aggregate?.aggregate?.count || 0;
     });
   }
+
+  getInfoNative() {
+    let nativeToken = this.tokenService.tokensMarket?.find(
+      (k) => k.coin_id === this.environmentService.coingecko?.ids[0],
+    );
+
+    const supplyAmount = balanceOf(this.tokenDetail.totalSupply, this.tokenDetail.decimals);
```

Comment:
Sử dụng `getBalance`
```suggestion
    const supplyAmount = getBalance(this.tokenDetail.totalSupply, this.tokenDetail.decimals);
```